jekyll/api.md

```diff
@@ -16,108 +16,140 @@ This value will be referenced below as `$APIKEY`.
 Open a local terminal, then run the following `curl` command:
 
 ```
-curl https://explorer.sustglobal.io/api/portfolios/?api_key=$APIKEY
+curl "https://explorer.sustglobal.io/api/portfolios/?api_key=$APIKEY"
```

Comment:
I added quotes around all the URLs in these examples to avoid any issues with escaping special characters on various terminals.